artman/cli/main.py

```diff
@@ -152,6 +152,15 @@ def parse_args(*args):
         'will first look up in artman user config. If not found, an error '
         'will be raised with instructions.',
     )
+    parser.add_argument(
+        '--toolkit-dir',
+        type=str,
+        default='',
+        help='[Optional] Local directory containing the local clone of the '
+        'github/toolkit repo to use, instead of the one specified in the '
+        'configuration files. This flag is useful for testing development '
+        'changes to toolkit via an artman invocation.',
+    )
```

Comment:
Were you suggesting to mount the local toolkit to the docker container and use that inside the docker container?

Reply:
Yes, exactly. I can mount via "-v .../toolkit/:/tmp/toolkit" to Docker and then run artman inside it with "--toolkit-dir /tmp/toolkit"

Reply:
In ~/.artman/config.yaml, there is local_paths.toolkit which will point artman to local toolkit dir. But that only works for local mode. If you are okay with mount via -v, you can simply update the ~/.artman/config.yaml inside the docker image without introducing this new flag.

Unlike --local-repo-dir that was recently added, I feel this --toolkit-dir will be even less used by normal users.

Reply:
Ah, OK. I couldn't this to work by changing the toolkit path in my non-Docker .artman.config.yaml. But changing it inside the Docker container works.
(We may want to think about propagating that information in automatically.)
I agree, in that case this PR is not necessary. I'll close it.